src/http/api/views/account.posts.view.integration.test.ts

```diff
@@ -44,55 +32,23 @@ describe('AccountPostsView', () => {
         // Clean up the database
         await fixtureManager.reset();
 
-        const siteData = {
-            host: 'example.com',
-            webhook_secret: 'secret',
-        };
-        const [id] = await db('sites').insert(siteData);
-
-        site = {
-            id,
-            ...siteData,
-        };
-
-        internalAccountData = {
-            username: 'index',
-            name: 'Test Site Title',
-            bio: 'Test Site Description',
-            avatar_url: 'https://example.com/avatar.jpg',
-            banner_image_url: 'https://example.com/banner.jpg',
-        };
-
         const logger = {
             info: vi.fn(),
         } as unknown as Logger;
 
         events = new AsyncEvents();
-        accountRepository = new KnexAccountRepository(db, events);
         postRepository = new KnexPostRepository(db, events, logger);
         const fedifyContextFactory = new FedifyContextFactory();
 
-        accountService = new AccountService(
-            db,
-            events,
-            accountRepository,
-            fedifyContextFactory,
-            generateTestCryptoKeyPair,
-        );
-
         viewer = new AccountPostsView(db, fedifyContextFactory);
 
         const [accountEntityTemp] =
             await fixtureManager.createInternalAccount();
         accountEntity = accountEntityTemp;
-        account = await db('accounts').where({ id: accountEntity.id }).first();
 
         const [siteDefaultAccountTemp] =
             await fixtureManager.createInternalAccount();
         siteDefaultAccount = siteDefaultAccountTemp;
-        defaultAccount = await db('accounts')
-            .where({ id: siteDefaultAccount.id })
-            .first();
```

Comment:
It doesn't look like we need `account` / `defaultAccount` in those tests, so looks like a valid cleanup!